### DISCUSSION TITLE
Docker image for 4.19

### DISCUSSION THREAD

**Discussion by fhoner** [pretix/pretix#3280] [General]
Hi,
I've just seen on your blog that version 4.19.0 was released. However I could not find a tag on dockerhub, the most recent one seems to be 4.18.2 or the post1 respectively? Is this expected or should it be there already?
Thanks, 
Felix

**Comment by raphaelm** [comment]
Thanks for the heads up, this is due to #3281 and we'll fix it and then push the new image.

**Reply by raphaelm** [reply]
Docker tags `4.19` and `4.19.0.post1` are now available
